runtime/Cpp/runtime/nuget/ANTLR4.Runtime.cpp.shared.nuspec

```
@@ -0,0 +1,30 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2012/06/nuspec.xsd">
+  <metadata>
+    <id>ANTLR4.Runtime.cpp.vs$vs$.shared</id>
+    <version>$version$$pre$</version>
+    <title>ANTLR4 Runtime c++ vs$vs$ shared</title>
+    <authors>Terence Parr &amp; Contributors</authors>
+    <requireLicenseAcceptance>true</requireLicenseAcceptance>
+    <license type='expression'>BSL-1.0</license>
+    <icon>image\antlr4.jpg</icon>
+    <projectUrl>https://www.antlr.org/</projectUrl>
+    <description>ANTLR (ANother Tool for Language Recognition) is a powerful parser generator for reading, processing, executing, or translating structured text or binary files. It's widely used to build languages, tools, and frameworks. From a grammar, ANTLR generates a parser that can build and walk parse trees..</description>
+    <releaseNotes>releaseNotes</releaseNotes>
+    <copyright>Copyright 2006-2020</copyright>
```

Probably the end year should be updated:

```suggestion
    <copyright>Copyright 2006-2022</copyright>
```

---

Fixed